phi/tools/firecrawl.py

```diff
@@ -49,16 +50,17 @@ def scrape_website(self, url: str) -> str:
         scrape_result = self.app.scrape_url(url, params=params)
         return json.dumps(scrape_result)
 
-    def crawl_website(self, url: str, limit: Optional[int] = None) -> str:
+    def crawl_website(self, url: str, limit: Optional[int] = None, wait_until_done: Optional[bool] = None) -> str:
```

Comment:
@Ayush0054 this should only be set on __init__, not on the tool call as this should be determined when initializing the toolkit